08_voting/index.tex

```diff
@@ -1 +1,101 @@
 \chapter{Voting}
+\section{Voting Scenarios}
+\label{sec:VotingScenarios}
+
+Within the context of the coursework, robust voting systems are designed for deciding the outcome of collective actions (e.g. elections for legislative, executive and judicial branches in IIGO).
+When voting is called in IIGO, the President, Speaker, Judge and the islands in the archipelago (i.e. the \emph{Voters}, assuming that no diplomatic sanction is in place) will be empowered to proceed with the following series of actions:
+\begin{itemize}
+    \item The President will be eligible to select a rule for the archipelago to vote for.
+    \item The Speaker will be eligible to take charge of enabling the voting procedure by receiving, counting ballots and ultimately, declaring the winner.
+    \item The islands in the archipelago (i.e. Voters) will be eligible to vote by submitting their ballots\footnote{This is based on the assumption that no diplomatic sanctions are in place for any islands.}.
+\end{itemize}
+
+In an IIGO session, the voting action is needed for deciding the next President, Speaker, Judge and also for deciding whether or not to adapt a new rule as well as to disable an active one.
```

Comment:
Different phrasal verb may be more appropriate. Suggested rewrite:

In an IIGO session, the voting action is needed to decide the next President, Speaker, Judge and also to decide whether or not to adapt a new rule as well as to disable an active one.

Reply:
designed for --> designed to .
ok I agree with this.
(same as above)

Reply:
changed